src/library/baseplaylistfeature.cpp

```diff
@@ -386,6 +383,47 @@ void BasePlaylistFeature::slotImportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotCreateImportPlaylist() {
+    if (!m_pPlaylistTableModel) return;
```

Comment:
Yes this is duplicated code but this is because the `PlaylistFeature` and the `CrateFeature` do not inherit from the same class. `PlaylistFeature` inherits from `BasePlaylistFeature` but `CrateFeature` only inherits from `LibraryFeature` and this last class does not have the data needed to create a playlist, a better option would be to inherit both `PlaylistFeature` and `CrateFeature` to avoid duplicated code but this can be a lot of work.